bin/wabur

```diff
@@ -129,15 +129,24 @@ options = {
 
 config = WAB::Impl::Configuration.new(usage, options)
 mode = config[:mode]
+# TBD: sanitize path to ensure ancestor-level directories cannot be written to.
+dir = File.expand_path(config[:base] || '.')
```

Comment:
There is no reason to sanitize the path. The user/developer is calling this in their own environment. If they want to pick a path above the Curren directory or anywhere on their machine that is up to them to decide.

Reply:
makes sense..